### PR TITLE
add some "cluster management" doc

### PR DESCRIPTION
**Pull request by kaspar030**

### CODE REVIEW

**Comment by kaspar030** [comment]
@cgundogan @aabadie @fjmolinas could you take a look at this?

File: README.md [hunk 1]
```diff
@@ -109,6 +109,108 @@ nginx example (in addition to SSL setup):
...
+ServerAliveCountMax 2
+```
+
+Make sure `~/.ssh/id_rsa_murdock-slave` can log in to `murdock-slave@ci.riot-os.org`.
+
+2. keep an ssh connection open that forwards the ports 7711 and 6379.
+
+E.g., use this alias and "autossh":
+
+    $ alias dwq_connect='autossh -M0 -N -C -f murdock
```
Comment (MrKevinWeiss) [inline README.md]:
```suggestion
    $ alias dwq_connect='autossh -M0 -N -C -f murdock'
```

File: README.md [hunk 2]
```diff
@@ -109,6 +109,108 @@ nginx example (in addition to SSL setup):
...
+
+## Setup
+
+1. set up ssh authentication to ci.riot-os.org.
+
+E.g., add this to `~/.ssh/config`:
+
+```
+Host murdock
+HostName ci.riot-os.org
```
Comment (MrKevinWeiss) [inline README.md:128]: Just to clarify, if we had another murdock master running, this would be the HostName of that (eg. `HostName test-ci.riot-os.org`)?

**Review by MrKevinWeiss — COMMENTED** [review]
This has been pretty useful to help me understand.

Assuming the comments are addressed I think it would be nice to have in.

**Comment by MrKevinWeiss** [comment]
I also managed to get it to work from my CPU

File: README.md (hunk 2 above)
Comment (kaspar030) [reply README.md:128]: Yes!

**Review by MrKevinWeiss — APPROVED** [review]
Followed instructions and it seems to work.  ACK! _(I don't have control of this repo though)_

**Comment by kaspar030** [comment]
Thanks for the review!

**Pull request merged**